### PR TITLE
updating scripts in package.json, enabling scripts to run single tests

### PR DESCRIPTION
**Pull request by andrew-sunada**
Updating script package.json names
Enabling scripts to run single tests
This includes a requirement github user access token within the .env file

### CODE REVIEW

**Review by mozafrank — COMMENTED** [review]
The code looks good and well-documented, one thing @andrew-sunada can you rename README.MD to README.md ?

I am a little concerned about how this change increases the scope of this tool. Having it download a file from github is a very specific use case, and not something it would ever need to do in prod, and would require the developer knowing the names of the checks they want, but not having a local copy of them, which seems like an edge case. I wonder if a simpler approach would be to just have "--test=" take a local path to a check. What do you think @andrew-sunada ?

**Comment by andrew-sunada** [comment]
> The code looks good and well-documented, one thing @andrew-sunada can you rename README.MD to README.md ?
> 
> I am a little concerned about how this change increases the scope of this tool. Having it download a file from github is a very specific use case, and not something it would ever need to do in prod, and would require the developer knowing the names of the checks they want, but not having a local copy of them, which seems like an edge case. I wonder if a simpler approach would be to just have "--test=" take a local path to a check. What do you think @andrew-sunada ?

Yea, I'm totally down for that. It does increase complexity and spec to pull it out and away. I wanted to have someone just be able to one line run it, but in the case that someone's using it, it wouldn't be hard to just copy pasta that. I'll set that up in here.

File: checkers/mozlenium/test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// EXAMPLE copied script
```
Comment (mozafrank) [inline checkers/mozlenium/test.js]: I'm ok with including examples but this should be clearly marked as doc, lets either make an examples directory in /checkers/mozlenium, or move this here, and replace demo-check.js: https://github.com/mozilla-it/mozalert/tree/main/examples

File: checkers/mozlenium/mozlenium-test-runner.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+
+/**
+ * Test runner that reads the script provided andtransforms it into something
+ * that will be useful for selenium.
+ */
+class MozleniumTestRunner {
+  static TEST_FILE = './test.js';
```
Comment (mozafrank) [inline checkers/mozlenium/mozlenium-test-runner.js]: if I'm interpreting this correctly, you're removing the need for https://github.com/mozilla-it/mozalert/blob/main/checkers/mozlenium/entrypoint.sh#L30-L32 which is excellent and this will make local dev a lot easier. To take this a step further it would be great to parameterize 'test.js' so the entrypoint script doesn't need to copy that into place. I could pass in the location of the check and mozlenium can handle it from there. does that seem legit @andrew-sunada ?

File: checkers/mozlenium/test.js (hunk 1 above)
Comment (andrew-sunada) [reply checkers/mozlenium/test.js]: I'm confused about what you mean here. 
- Make an examples directory. Check.
- remove "check.js" in favour of "demo-check.js"
- the rest is confusing to me

File: checkers/mozlenium/mozlenium-test-runner.js (hunk 2 above)
Comment (andrew-sunada) [reply checkers/mozlenium/mozlenium-test-runner.js]: Yes, I'm removing the need for that entrypoint.sh line set. Good! I'm glad it's better. I wasn't sure if bash's write functionality was faster/more efficient than js's. 

Yea, for sure! I can parameterize that. Thanks for that feedback!
